testfiles/1ZoneUncontrolled_9161.idf

```diff
@@ -0,0 +1,461 @@
+!-Generator IDFEditor 1.34
```

Comment:
As-is, this will result in this file showing up in installer packages.  It can be excluded by prefixing an underscore on the file name, or explicitly listing it in the exclude patterns in `cmake/Install.cmake`.

Looking forward, if we wanted to use the same example file but for multiple weather files, we could add a new argument to the `add_simulation_test` function that would add a suffix to the ctest entry to make it unique.  That's obviously too much for this little PR, but could be done pretty easily.

For now, unless there is a reason not to, I'll plan on adding an underscore to the filename and we'll wrap this up.  

Thanks for adding this test @mbadams5

Reply:
@Myoldmopar, sure that is fine to just add the underscore and move on for this PR. I figured it would add it to the installer but I forgot how to avoid that. The additional argument on the `add_simulation_test` would be good since I tried to do a duplicate test at first.